### DISCUSSION TITLE
Panel data grid has incorrect filters for UUID

### DISCUSSION THREAD

**Discussion by iBobik** [contember/admin#551] [Support]
UI for UUID field allows to use contains filter, which will brick the view because filters are also persisted in the localstorage.

<img width="810" alt="Snímek obrazovky 2023-06-25 v 2 04 29" src="https://github.com/contember/interface/assets/614232/a39fd4fb-a060-4ef9-b3f4-8fdc043f98ef">
<img width="1314" alt="Snímek obrazovky 2023-06-25 v 2 03 54" src="https://github.com/contember/interface/assets/614232/9910c72f-deb4-4150-8f97-05f411b7447f">

Btw, I would not need to use contains filter if if would be possible to easily copy full UUID value.

**Comment by honzasladek** [comment]
Good catch on both parts! I'll create an issue for both, the first is definitely a bug.
